sw/device/tests/dif/dif_plic_unittest.cc

```diff
@@ -373,5 +373,90 @@ TEST_F(IrqCompleteTest, Target0Success) {
   }
 }
 
+class SoftwareIrqForceTest : public PlicTest {
+ protected:
+  // Make sure our tests are up to date.
+  SoftwareIrqForceTest() { EXPECT_EQ(RV_PLIC_PARAM_NUMTARGET, 1); }
```

Comment:
`static_assert`?

Reply:
I think this will be changed as soon as @mcy renaming lands anyway.

Reply:
It is a `static_assert` now.